src/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import './PeoplePage.scss';
+import { useLocation, useNavigate } from 'react-router-dom';
+import debounce from 'lodash/debounce';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../api/getPeople';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const fetchPeople = async () => {
+    const peopleFromServer = await getPeople();
+
+    setPeople(peopleFromServer);
+  };
+
+  useEffect(() => {
+    fetchPeople().then(r => r);
+  }, []);
+
+  let updatePeople = people.map(person => (
+    {
+      ...person,
+      motherNameSlug: people.find(mother => person.motherName === mother.name),
+      fatherNameSlug: people.find(father => person.fatherName === father.name),
+    }
+  ));
```

Comment:
wrap this inside useMemo hook
https://dmitripavlutin.com/react-usememo-hook/
```suggestion
  let updatedPeople = people.map(person => (
    {
      ...person,
      motherNameSlug: people.find(mother => person.motherName === mother.name),
      fatherNameSlug: people.find(father => person.fatherName === father.name),
    }
  ));
```